CHANGELOG.md

```diff
@@ -66,7 +66,7 @@ A brief description of the categories of changes:
   bootstrap .py file when using {obj}`--bootstrap_impl=script`
 * (toolchain) The {bzl:obj}`gen_python_config_settings` has been fixed to include
   the flag_values from the platform definitions.
-
+* (whl_filegroup): Provide per default also the `RECORD` file
```

Comment:
Could you please rebase and update the `CHANGELOG.md` to point to things added in an unreleased version?

Reply:
hope it is now where you want it :-)